src/registrar/templates/emails/status_change_in_review.txt

```diff
@@ -0,0 +1,81 @@
+{% autoescape off %}{# In a text file, we don't want to have HTML entities escaped #}
+Hi {{ application.submitter.first_name }}.
+
+Your .gov domain request is being reviewed.
+
+DOMAIN REQUESTED: {{ application.requested_domain.name }}
+REQUEST RECEIVED ON: {{ application.updated_at|date }}
+REQUEST #: {{ application.id }}
+STATUS: In review
+
+
+NEED TO MAKE CHANGES?
+
+If you need to change your request you have to first withdraw it. Once you
+withdraw the request you can edit it and submit it again. Changing your request
+might add to the wait time. Learn more about withdrawing your request.
+<https://get.gov/help/domain-requests/#withdraw-your-domain-request>.
+
+
+NEXT STEPS
+
+- We’re reviewing your request. This usually takes 20 business days. 
+
+- You can check the status of your request at any time. 
+  <https://registrar.get.gov/application/{{ application.id }}>
+
+- We’ll email you with questions or when we complete our review.
+
+
+THANK YOU
+
+.Gov helps the public identify official, trusted information. Thank you for
+requesting a .gov domain.
+
+----------------------------------------------------------------
+
+SUMMARY OF YOUR DOMAIN REQUEST
```

Comment:
**suggestion**: Since we have access to the power of Django's templates here in our emails, I think that we should make a separate include file for this domain summary so that we can just `{% include emails/includes/application_summary.html %}` here and in `templates/emails/submission_confirmation.txt`. That way, if we fix or improve the summary section we don't have to follow the copy-paste and repeat the fixes.